frontend/src/lib/components/PropertyGroupFilters/PropertyGroupFilters.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { CSSProperties, useEffect } from 'react'
+import { useValues, BindLogic, useActions } from 'kea'
+import '../../../scenes/actions/Actions.scss'
+import { AndOrPropertyFilter, AndOrPropertyGroup } from '~/types'
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { Button, Col, Row, Select } from 'antd'
+import { PlusOutlined, DeleteOutlined } from '@ant-design/icons'
+import './PropertyGroupFilters.scss'
+import { propertyGroupFilterLogic } from './propertyGroupFilterLogic'
+import { PropertyFilters } from '../PropertyFilters/PropertyFilters'
+
+interface PropertyGroupFilters {
+    propertyFilters?: AndOrPropertyFilter | null
+    onChange: (filters: AndOrPropertyFilter) => void
+    pageKey: string
+    style?: CSSProperties
+    taxonomicGroupTypes?: TaxonomicFilterGroupType[]
+    eventNames?: string[]
+}
+
+export function PropertyGroupFilters({
+    propertyFilters = null,
+    onChange,
+    pageKey,
+    taxonomicGroupTypes,
+    style = {},
+    eventNames = [],
+}: PropertyGroupFilters): JSX.Element {
+    const logicProps = { propertyFilters, onChange, pageKey }
```

Comment:
Tip, if you need to look up this logic by key later, here's a pattern that can be used to avoid passing in a unique `pageKey` to the component:

https://github.com/PostHog/posthog/blob/ba640115e91bdab596cfb3c221d05de66b0e844a/frontend/src/lib/components/ObjectTags/ObjectTags.tsx#L54